### PR TITLE
macOS name fix

### PR DESCRIPTION
**Pull request by duncdrum**
Since 2016  “Mac OS X” is now macOS. Please squash or rebase on merge.

### CODE REVIEW

File: data/troubleshooting.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
 06 Oct 2012 16:56:32,824 [main] INFO  (JettyStart.java [run]:129) - [eXist Build : 20121006] 
 06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:131) - [eXist Home : unknown] 
 06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:133) - [SVN Revision : 17031] 
-06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:141) - [Operating System : Mac OS X 10.8.2 x86_64] 
+06 Oct 2012 16:56:32,825 [main] INFO  (JettyStart.java [run]:141) - [Operating System : macOS 10.8.2 x86_64]
```
Comment (joewiz) [inline data/troubleshooting.xml]: This one should probably stay as is.

File: data/webdav.xml [hunk 2]
```diff
@@ -95,8 +95,8 @@
                 </para>
             </section>
             <section id="macosx">
```
Comment (joewiz) [inline data/webdav.xml]: Perhaps the section ID should also lose the `x`, to be just `macos`.

**Comment by duncdrum** [comment]
good catches, done.

**Comment by joewiz** [comment]
@duncdrum Thanks! We're in a code freeze on the core repositories included in eXist installer packages, but will merge your PR once 3.1.0 is released.

**Comment by shabanovd** [comment]
@joewiz good to merge?

**Pull request merged**